src/Tizen.NUI.Components/Controls/ViewList.cs

```diff
@@ -0,0 +1,302 @@
+            {
+                return internalViewList.Contains(view);
+            }
+
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public void CopyTo(View[] array, int arrayIndex)
+            {
+            }
+            /// <summary>
+            /// Remove a View from the list.
+            /// </summary>
+            /// <param name="view">The to-be-removed View object</param>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public bool Remove(View view)
+            {
+                if (!internalViewList.Contains(view)) return false;
+                RemoveAt(internalViewList.IndexOf(view));
+                return true;
+            }
+            /// <summary>
+            /// Get the enumerator of the list.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public IEnumerator<View> GetEnumerator()
+            {
+                return internalViewList.GetEnumerator();
+            }
+            /// <summary>
+            /// Get the enumerator of the list.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            IEnumerator IEnumerable.GetEnumerator()
+            {
+                return internalViewList.GetEnumerator();
+            }
+        }
+    }
+
+    /// <summary>
+    /// Frame is a class which keeps Views in a frame and
+    /// shows one selected View.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class Frame : ViewList
+    {
+        private View frameView;
+
+        /// <summary>
+        /// Creates a new instance of Frame.
+        /// </summary>
+        public Frame()
+        {
+            frameView = new View();
+            frameView.WidthSpecification = LayoutParamPolicies.MatchParent;
+            frameView.HeightSpecification = LayoutParamPolicies.MatchParent;
+            this.Add(frameView);
+        }
+
+        /// <summary>
+        /// Insert a View to the Frame.
+        /// </summary>
+        /// <param name="order"></param>
+        /// <param name="tab"></param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override void Insert(int order, View view)
+        {
+            view.WidthSpecification = LayoutParamPolicies.MatchParent;
+            view.HeightSpecification = LayoutParamPolicies.MatchParent;
+        }
+
+        /// <summary>
+        /// Remove a View from the Frame.
+        /// </summary>
+        /// <param name="order">The order of the to-be-removed object</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override void RemoveAt(int order)
+        {
+        }
+
+        /// <summary>
+        /// Select one object from the Frame.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override void Select(int index)
+        {
+            if (index >= Views.Count) return;
```

Comment:
Please throw exception for unexpected behavior.
```suggestion
            if (index >= Views.Count) throw new ArgumentOutOfRangeException(nameof(index), [message]);
```